src/components/navbar.js

```diff
@@ -0,0 +1,23 @@
+import {LogStatusContext} from "./Contexts/LogStatusContext";
+import Logo from "../assets/logo.png";
+import React from "react";
+import {Link} from "react-router-dom";
+
+export const NavBar = () => (
+    <LogStatusContext.Consumer>
+        {({status, toggleLogStatus}) => (
+            <nav className="navbar navbar-expand-lg justify-content-between">
+                <Link className="navbar-brand" to="/">
+                    <img src={Logo} height="50" alt=""/>
+                </Link>
+                <Link to="/about" className={`navbar-btn ${status ? "" : "hidden"}`}>
+                    About
+                </Link>
+                <Link onClick={toggleLogStatus}
```

Comment:
the idea of `toggleLogStatus` doesn't smell good to me 🤔

What this function is doing is: "I don't care about what the current value is, just switch it" and that's not good. Why? well, this flexibility could lead to bugs, since you don't have control about the actions that are updating the state. That function could be misused in any other place in the code. What If I can call `toggleLogStatus` different times?

I would rather have an specific `logoutAction` that set the `logStatus` to false, that way you don't need to care how many times that function is called, it'll always produce the same status

Reply:
I see you point , I reworked it to make it predictable as suggested